### ISSUE TITLE
Screen size does not work

### ISSUE THREAD

**Issue by hendrikbez** [dougmencken/HeadOverHeels#76]
I am trying to make the screen bigger, but it does not work
![Screenshot_77](https://github.com/user-attachments/assets/05174b78-d607-4f6e-b73c-1e1339d0a716)
![Screenshot_76](https://github.com/user-attachments/assets/5f6ece58-044a-401c-88ca-b02abc9f771c)

**Comment by dougmencken** [comment]
yeah, currently slides aren’t getting recreated, in 9ccc9774e68ae43365a28e8f4eeadcc255744de4 I just added the menu, but I’m dealing with it

**Comment by dougmencken** [comment]
by the way, I think about providing 2× pixels for sizes greater than 1280×960, when each pixel will be not 1×1 but 2×2

**Comment by hendrikbez** [comment]
Good morning.   

I do not know if you have fix the size, just downloaded the new version from you site and it still not show full screen.
![head over heels 2](https://github.com/user-attachments/assets/ba2cb77a-ce57-48a2-9e17-580d3218ae86)
![head over heels](https://github.com/user-attachments/assets/e0472be7-1f64-4ea3-b499-80ab16eed34f)

**Comment by dougmencken** [comment]
@hendrikbez it works well on my side

![Screenshot from 2024-10-03 07-53-44](https://github.com/user-attachments/assets/66e542ee-bf72-4318-b699-cc2ee07eda3a)
